### DISCUSSION TITLE
Delay in useColorModeValue()

### DISCUSSION THREAD

**Discussion by slxsh** [chakra-ui/chakra-ui#6141] [Q&A] [answered]
I have a navbar and use the `useColorModeValue` hook to set its background colour. The problem is this hook returns the light colour value (#FFFFFF) regardless of the light/dark theme. This results in the navbar being white for a split second and then turning to dark mode colour.
```
const bgColor = useColorModeValue("#FFFFFF", "#1A202C")

// using this value like this
<Flex 
            direction={"row"} 
            w={"100%"} 
            pos={"sticky"}
            top={0} 
            p={3} 
            borderBottomWidth={0}
            justifyContent={"space-between"}
            alignItems={"center"}
            px={10}
            bg={bgColor}
            zIndex={5}
            boxShadow={shadow}
>
```

**Comment by aroyan** [comment, marked as answer]
Are you referring to this? [Color Mode Flash Issue](https://chakra-ui.com/docs/styled-system/features/color-mode#color-mode-flash-issue). It's known bug afaik and it's not fixed yet.
